lib/taurus/qt/qtgui/base/taurusbase.py

```diff
@@ -1350,13 +1371,27 @@ def showFormatterDlg(self):
         (in string version) or None
         """
         current_format = self.getFormat()
+        ind = self._formaters.values().index(current_format)
+
+        formatter, ok = Qt.QInputDialog.getItem(self, "Set formatter",
```

Comment:
Since the dialog does not allow to customize the tooltip, use the "label text" to give the hint (maybe using an html link to the formatter docs?)